GpsApp/Gps/GpsComponentImpl.cpp

```diff
@@ -21,10 +21,16 @@ namespace GpsApp {
   // ----------------------------------------------------------------------
 
   GpsComponentImpl ::
+#if FW_OBJECT_NAMES == 1
```

Comment:
This FW_OBJECT_NAMES is a pattern we decided to remove (though it looks like we forgot to update the GPS app tutorial!).

Instead we always provide a component name but substitute it with an empty string  if object names are disabled. You can see some example of this in F' Ref app.